build-lib/lib.js

```diff
@@ -95,12 +89,13 @@ window.ShopifyBuy = require('shopify-buy/shopify').default;
     const minifiedTree = uglifyJavaScript(funnel(tree, {
       getDestinationPath: function (path) {
         return path.replace(/\.js/, '.min.js');
-      }
+      },
+      exclude: ['**/*.map']
     }));
 
-    const concatenatedScripts = new Licenser([mergeTrees([tree, minifiedTree])]);
+    const licensedScripts = new Licenser([mergeTrees([tree, minifiedTree], { overwrite : true })]);
 
-    tree = mergeTrees([concatenatedScripts, nodeLibOutput]);
+    tree = mergeTrees([tree, minifiedTree, licensedScripts, nodeLibOutput], { overwrite : true });
```

Comment:
That's correct. In this case, `Licenser` prepends the LICENSE to the top of each JS file fed into it (both minified & originals) and then generates new files with the same name.

The `overwrite` config will make the licensed JS files overwrite the non-licensed ones. Meanwhile, maps generated from `concat` above are left untouched.